### ISSUE TITLE
Device incompatibility: everything rotated by 90 degree on Gemini PDA

### ISSUE THREAD

**Issue by JonasT** [wireapp/wire-android#1873]
## The problem

Everything appears rotated by 90 degree on Gemini PDA, making the app unusable on that device.
Here is how the device looks like, just so you understand the form factor:

![](https://i.ytimg.com/vi/bYur037PT_M/maxresdefault.jpg)

The device runs Android and has a phone-sized screen, and as you can easily see, landscape mode is just not optional for anything running on this. I'm aware there are tickets for this, I just wanted to bring to your attention that for some devices, this is more than "a bit cumbersome when I want to use it with a bluetooth keyboard".

## Environment

* Wire version that exhibits the issue: 3.21.661
* Last Wire version that did not exhibit the issue (if applicable):
* Android OS version used to run Wire: 7.1.1
* Is this a custom firmware or a stock one: stock but rooted (as provided by vendor)
* Mobile phone model/manufacturer: Planet Computers
* Mobile network type (EDGE/LTE/Wi-Fi/Offline): all affected

**Comment by therealpxc** [comment]
I use Wire for all of my group chats, making this a really killer bug for me.

Sometimes, in between screens, things will temporarily appear in a landscape layout. It taunts me. :'(

Thanks @JonasT for reporting

**Comment by therealpxc** [comment]
Hey, I've since noticed that Wire on Android operates on my Gemini in landscape mode!

For what it's worth, I've found it very useable, although I miss keyboard shortcuts from the desktop version.

I don't think I enabled any relevant hacks for my device. In case landscape mode has indeed been enabled in the app, thank you very much! I'm enjoying the ability to use my favorite chat app so fluently on the go.

@JonasT are you still experiencing issues?

**Comment by JonasT** [comment]
Yes, I do. And I'm pretty sure planet provides a custom hack to force landscape mode, but I'm not using the app bar or other software, and it shouldn't be required. Wire really should just be fixed.
